addons/web_editor/static/src/js/wysiwyg/widgets/media.js

```diff
@@ -104,6 +105,16 @@ var SearchableMediaWidget = MediaWidget.extend({
     // Handlers
     //--------------------------------------------------------------------------
 
+    /**
+     * @private
+     */
+    _onSearchKeydown: function (ev) {
+        // If the template contains a form that has only one input, the enter
+        // will reload the page as the html 2 specify this behavior.
+        if (ev.originalEvent && ev.originalEvent.code === "Enter") {
```

Comment:
Does it work in mobile ? Doesn't it reload anyway ? Maybe it doesn't reload in mobile even in current stable ?

Reply:
The `originalEvent.code` was undefined but `ev.originalEvent.key` is `Enter`.
I've tested it and it works in mobile.